vdj_ann/src/annotate.rs

```diff
@@ -3050,6 +3051,7 @@ impl ContigAnnotation {
         invalidated_umis: Option<Vec<String>>,   // invalidated UMIs
         is_cellx: bool,                          // was the barcode declared a cell?
         productivex: bool,                       // productive?
+        productive_criteria: ContigStatus,       // criteria used to asess productive status
```

Comment:
Same kind of thing here.
```suggestion
        /// Criteria used to asess productive status.
        productive_criteria: ContigStatus,
```